### ISSUE TITLE
Verify focus ring colour in GEL docs

### ISSUE THREAD

**Issue by samithaf** [WestpacGEL/GEL-next#571]
![Screenshot 2023-12-15 at 1 27 52 pm](https://github.com/WestpacGEL/GEL-next/assets/1860417/e0f5925e-b0cd-4617-abde-77a39047232f)
![Screenshot 2023-12-15 at 1 28 05 pm](https://github.com/WestpacGEL/GEL-next/assets/1860417/363c748c-a592-482f-b993-440ade2c56bb)

**Comment by KateMacdonald** [comment]
@samithaf Each brand should use it's respective focus colour - called _'Focus'_ in the multi-brand palette. 

PS The colour swatches for this are missing from the Accessibility tab of the colour page - i will add a new issue.

**Comment by samithaf** [comment]
Fixed. @KateMacdonald please check again.
